autoload/neomake.vim

```diff
@@ -645,7 +648,22 @@ function! s:AddExprCallback(jobinfo, prev_index) abort
         let index += 1
 
         let before = copy(entry)
-        call call(s:postprocess, [entry], maker)
+        if type(s:postprocess) == type([])
+            let s = s:postprocess
+            for s:f in s:postprocess
+                if type(s:f) == type({})
+                    let s:this = extend(copy(s:f), {'maker': maker})
+                    call call(s:f.fn, [entry], s:this)
+                else
+                    let s:this = {'maker': maker}
+                    call call(s:f, [entry], s:this)
+                endif
+            endfor
+            " TODO: abort on first invalid entry already?!
```

Comment:
> Since it's a list of processors now, passing through `entry != before` might help debug a processor that was affected by a prior processor.

You mean to set this on `s:this`/`self`, e.g. `s:this.changed = entry != before`?
We could also pass in `before` as `orig_entry` then instead.